### PR TITLE
Apostrophe to Single quote

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: pages/agent/v3/cli_step.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # buildkite-agent step
 
-The Buildkite Agent’s `step` command provides the ability to retrieve and  update the attributes of steps in your `pipeline.yml` files.
+The Buildkite Agent's `step` command provides the ability to retrieve and  update the attributes of steps in your `pipeline.yml` files.
```
Comment (karensawrey) [inline pages/agent/v3/cli_step.md.erb:3]:
```suggestion
The Buildkite agent's `step` command provides the ability to retrieve and update the attributes of steps in your `pipeline.yml` files.
```

File: pages/agent/v3/gcloud.md.erb [hunk 2]
```diff
@@ -41,7 +41,7 @@ Fetching cluster endpoint and auth data.
 kubeconfig entry generated for YOUR-CLUSTER-NAME
 ```
 
-Double check it’s running by taking a look at the cluster info:
+Double check it's running by taking a look at the cluster info:
```
Comment (karensawrey) [inline pages/agent/v3/gcloud.md.erb:44]:
```suggestion
Double check that it's running by taking a look at the cluster info:
```

File: pages/agent/v3/gcloud.md.erb [hunk 3]
```diff
@@ -146,7 +146,7 @@ $ kubectl logs buildkite-agent-67d54b9b88-jnzxg buildkite
 2021-05-26 12:55:27 INFO   buildkite-agent-67d54b9b88-jnzxg-1 Waiting for work...
 ```
 
-You’re successfully running a Buildkite Agent!
+You're successfully running a Buildkite Agent!
```
Comment (karensawrey) [inline pages/agent/v3/gcloud.md.erb:149]:
```suggestion
You're successfully running a Buildkite agent!
```

File: pages/agent/v3/github_ssh_keys.md.erb [hunk 4]
```diff
@@ -3,29 +3,29 @@
...
 </div>
 
 {:toc}
 
 ## Method 1: Machine user
 
-Creating a [machine user](https://developer.github.com/guides/managing-deploy-keys/#machine-users) is the simplest way to create a single SSH key which provides access to your organization’s repositories.
+Creating a [machine user](https://developer.github.com/guides/managing-deploy-keys/#machine-users) is the simplest way to create a single SSH key which provides access to your organization's repositories.
 
 The easiest way to do this is by creating a “Buildkite Agent” machine user in your organization, and adding it to a team that has access to the relevant repositories.
```
Comment (karensawrey) [inline pages/agent/v3/github_ssh_keys.md.erb:15]:
```suggestion
The easiest way to do this is by creating a “Buildkite agent” machine user in your organization and adding it to a team that has access to the relevant repositories.
```

File: pages/agent/v3/prioritization.md.erb [hunk 5]
```diff
@@ -1,11 +1,11 @@
 # Buildkite Agent Prioritization
 
-By setting an Agent’s priority value you determine when it gets assigned build jobs compared to other Agents.
+By setting an Agent's priority value you determine when it gets assigned build jobs compared to other Agents.
```
Comment (karensawrey) [inline pages/agent/v3/prioritization.md.erb:3]:
```suggestion
By setting an Agent's priority value you determine when it gets assigned build jobs compared to other agents.
```

File: pages/agent/v3/prioritization.md.erb [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
 Agents with a higher value priority number are assigned work first, with the last priority being given to Agents with the default value of `null`.
 
 {:notoc}
 
-To set an Agent’s priority you can set it in the configuration file:
+To set an Agent's priority you can set it in the configuration file:
```
Comment (karensawrey) [inline pages/agent/v3/prioritization.md.erb:8]:
```suggestion
To set an agent's priority you can set it in the configuration file:
```

File: pages/agent/v3/ssh_keys.md.erb [hunk 7]
```diff
@@ -6,7 +6,7 @@ If your agent needs to clone your repositories using git and SSH, you'll need to
 
 ## Finding your SSH key directory
 
-When the Buildkite Agent runs any git operations, SSH will look for keys in `~/.ssh` under whichever user the agent runs as. Each platform’s [agent installation documentation](/docs/agent/v3/installation) specifies the exact location of this directory.
+When the Buildkite Agent runs any git operations, SSH will look for keys in `~/.ssh` under whichever user the agent runs as. Each platform's [agent installation documentation](/docs/agent/v3/installation) specifies the exact location of this directory.
```
Comment (karensawrey) [inline pages/agent/v3/ssh_keys.md.erb:9]:
```suggestion
When the Buildkite agent runs any git operations, SSH will look for keys in `~/.ssh` under whichever user the agent runs as. Each platform's [agent installation documentation](/docs/agent/v3/installation) specifies the exact location of this directory.
```

File: pages/agent/v3/tokens.md.erb [hunk 8]
```diff
@@ -10,9 +10,9 @@ When you create a new organization in Buildkite, a default agent token is create
 
 ## Using and storing tokens
 
-The token is used by the Buildkite Agent’s [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
+The token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
```
Comment (karensawrey) [inline pages/agent/v3/tokens.md.erb:13]:
```suggestion
The token is used by the Buildkite agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
```

File: pages/integrations/artifactory.md.erb [hunk 9]
```diff
@@ -1,12 +1,12 @@
 # Artifactory
 
-There are many ways to use [Artifactory](https://jfrog.com/artifactory/) with Buildkite. This document covers how to configure the Buildkite Agent’s built-in Artifactory support, as well as how to use Artifactory’s package management features in your Buildkite pipelines.
+There are many ways to use [Artifactory](https://jfrog.com/artifactory/) with Buildkite. This document covers how to configure the Buildkite Agent's built-in Artifactory support, as well as how to use Artifactory's package management features in your Buildkite pipelines.
 
 {:toc}
 
-## Buildkite Agent’s Artifactory support
+## Buildkite Agent's Artifactory support
```
Comment (karensawrey) [inline pages/integrations/artifactory.md.erb:7]:
```suggestion
## Buildkite agent's Artifactory support
```

File: pages/integrations/artifactory.md.erb [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
 
-The Buildkite Agent can upload and download artifacts directly from Artifactory. Export the following environment variables in your [Agent environment hook](/docs/agent/v3/hooks) to configure the Agent’s Artifactory support.
+The Buildkite Agent can upload and download artifacts directly from Artifactory. Export the following environment variables in your [Agent environment hook](/docs/agent/v3/hooks) to configure the Agent's Artifactory support.
```
Comment (karensawrey) [inline pages/integrations/artifactory.md.erb:9]:
```suggestion
The Buildkite agent can upload and download artifacts directly from Artifactory. Export the following environment variables in your [agent environment hook](/docs/agent/v3/hooks) to configure the agent's Artifactory support.
```

File: pages/integrations/bitbucket.md.erb [hunk 11]
```diff
@@ -48,7 +48,7 @@ Buildkite prompts you to give permission for Buildkite to post status updates, t
 
 ## Mercurial support
 
-Buildkite Agent’s default source control tool is Git, and doesn’t support [Mercurial](https://www.mercurial-scm.org) out of the box. If you want to use
+Buildkite Agent's default source control tool is Git, and doesn't support [Mercurial](https://www.mercurial-scm.org) out of the box. If you want to use
```
Comment (karensawrey) [inline pages/integrations/bitbucket.md.erb:51]:
```suggestion
Buildkite agent's default source control tool is Git, and doesn't support [Mercurial](https://www.mercurial-scm.org) out of the box. If you want to use
```

File: pages/pipelines/artifacts.md.erb [hunk 12]
```diff
@@ -1,8 +1,8 @@
 # Using Build Artifacts
 
-In this guide we’ll walk through using the Buildkite Agent’s [artifact support](/docs/agent/v3/cli-artifact)
+In this guide we'll walk through using the Buildkite Agent's [artifact support](/docs/agent/v3/cli-artifact)
```
Comment (karensawrey) [inline pages/pipelines/artifacts.md.erb:3]:
```suggestion
In this guide, we'll walk through using the Buildkite agent's [artifact support](/docs/agent/v3/cli-artifact)
```

File: pages/pipelines/artifacts.md.erb [hunk 13]
```diff
@@ -83,4 +83,4 @@ To further tighten the security in a Buildkite organization, you can use the [AP
 
 ## Further documentation
 
-See the [Buildkite Agent artifact documentation](/docs/agent/v3/cli-artifact) for a full list of options and details of Buildkite’s artifact support.
+See the [Buildkite Agent artifact documentation](/docs/agent/v3/cli-artifact) for a full list of options and details of Buildkite's artifact support.
```
Comment (karensawrey) [inline pages/pipelines/artifacts.md.erb:86]:
```suggestion
See the [Buildkite agent artifact documentation](/docs/agent/v3/cli-artifact) for a full list of options and details of Buildkite's artifact support.
```

File: pages/pipelines/build_meta_data.md.erb [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # Using Build Meta-data
 
-In this guide we’ll walk through using the Buildkite Agent’s [meta-data command](/docs/agent/v3/cli-meta-data) to store and retrieve data between different steps in a build pipeline.
+In this guide we'll walk through using the Buildkite Agent's [meta-data command](/docs/agent/v3/cli-meta-data) to store and retrieve data between different steps in a build pipeline.
```
Comment (karensawrey) [inline pages/pipelines/build_meta_data.md.erb:3]:
```suggestion
In this guide, we'll walk through using the Buildkite agent's [meta-data command](/docs/agent/v3/cli-meta-data) to store and retrieve data between different steps in a build pipeline.
```

File: pages/pipelines/build_meta_data.md.erb [hunk 15]
```diff
@@ -59,4 +59,4 @@ For more information see the [Buildkite REST Builds API documentation](https://b
 
 ## Further documentation
 
-See the [Buildkite Agent build meta-data documentation](/docs/agent/v3/cli-meta-data) for a full list of options and details of Buildkite’s meta-data support.
+See the [Buildkite Agent build meta-data documentation](/docs/agent/v3/cli-meta-data) for a full list of options and details of Buildkite's meta-data support.
```
Comment (karensawrey) [inline pages/pipelines/build_meta_data.md.erb:62]:
```suggestion
See the [Buildkite agent build meta-data documentation](/docs/agent/v3/cli-meta-data) for a full list of options and details of Buildkite's meta-data support.
```

**Review by karensawrey — COMMENTED** [review]
I little bit of additional combing while I'm at it.

**Review by karensawrey — APPROVED** [review]
Now it LGTM. Let's ship it!

**Comment by lachlanhardy** [comment]
@karensawrey I made a tiny commit to fix the failing test for you

**Pull request merged**